packages/flutter/lib/src/material/data_table.dart

```diff
@@ -408,6 +410,14 @@ class DataTable extends StatelessWidget {
   /// This value defaults to 56.0 to adhere to the Material Design specifications.
   final double columnSpacing;
 
+  /// Whether the widget should display checkboxes for selectable rows.
+  ///
+  /// If true, a [CheckBox] will be placed at the beginning of each row that is
+  /// selectable.
+  ///
+  /// If false, all rows will not display a [CheckBox].
+  final bool showCheckboxColumn;
```

Comment:
this should say something about how even if it's true, the column won't be included if there's no row with an `onSelectChanged` callback set.

Reply:
Done.